java/code/src/com/redhat/rhn/frontend/action/token/ActivationKeyDetailsAction.java

```diff
@@ -331,7 +331,9 @@ private ActivationKey create(DynaActionForm daForm, RequestContext context) {
 
     private Channel lookupChannel(DynaActionForm daForm, User user) {
         Long selectedChannel = (Long)daForm.get(SELECTED_BASE_CHANNEL);
-
+        if (selectedChannel == null) {
+            throw new ValidatorException("No Base Channel selected");
```

Comment:
Is this message shown in the page and should we localized it? According to the `ValidatorException` constructor usages, I see that currently  sometimes we do and sometimes we don't...

Reply:
I added the translation